src/components/table-transactions/table-transactions.js

```diff
@@ -43,6 +48,20 @@ export class TableTransactionsComponent {
     );
   }
 
+  mountChildren() {
+    this.confirmDialog = new ConfirmDialogComponent(
+      this.confirmDialogMountPoint,
+      {
+        onOkClick: this.handleOkClick.bind(this)
+      }
+    );
+    this.confirmDialog.mount();
+  }
+
+  handleOkClick(elem) {
```

Comment:
It is not intuitive name of handler. we have many ok buttons. Name it like `confirm delete` or smth like this.